src/components/building/buildingcard.js

```diff
@@ -120,16 +136,18 @@ const StyledBuildInfoContainer = styled.article`
   border: 1px solid black;
   display: flex;
   flex-direction: column;
-  padding: 5px;
+  min-height: 158px;
+  margin: 0;
 `;
 
 const StyledSpan = styled.span`
   display: flex;
   justify-content: space-between;
```

Comment:
You shouldn't use a `span` as a `flex` container as you usually expect a span to have `display: inline`

Reply:
with inline, it doesn't look like it should. but since it's part of the style, i'll postpone the whole thing until thursday, when i'll really start with style.

![image](https://user-images.githubusercontent.com/77203530/183404188-b95b8bed-f9ec-4325-b7b0-881b92762e66.png)

Reply:
I think you got me wrong: You should use `flex` and `space-between` but I think it's more common to use a `div` instead of a `span` for that.